superbuild/projects_modules/music-plugins.cmake

```diff
@@ -29,13 +29,11 @@ function(music_plugins_project)
     if (NOT USE_SYSTEM_${external_project})
 
         set(git_url ${GITHUB_PREFIX}Inria-Asclepios/music.git)
-        set(git_tag music3)
-
-        set(${external_project}_BUILD_TYPE RelWithDebInfo CACHE STRING "Build type for MUSICardio plugins: None Debug Release RelWithDebInfo MinSizeRel")
+        set(git_tag mm-simu)
```

Comment:
juste before the merge of this PR i'll change that branch to music3. For now, you can test this PR if you want with mm-simu